topics/proteomics/tutorials/biomarker_selection/tutorial.md

```diff
@@ -0,0 +1,376 @@
+>   {: .comment}
+{: .hands_on}
+
+We will now rerun the same tool but to select transcripts according to their expression profile.
+
+
+> ### {% icon hands_on %} Hands-on: Build tissue-specific expression dataset based on RNAseq
+>
+> 1. **Build tissue-specific expression dataset** {% icon tool %} with the following parameters:
+>    - *"Experimental data source (antibody- or RNAseq-based)"*: `RNA levels based on RNA-seq experiments`
+>    - *"Select tissue"*: `Heart muscle`
+>
+>   > ### Output
+>   > - **Tissue-specific expression from RNAseq** (19613 lines): List of the selected transcripts.
+>   > 4 columns: 'Gene', 'Gene name' and the retrieved info from HPA.
+>   {: .comment}
+{: .hands_on}
+
+
+This second list must be reduced by removing transcripts that are not highly enriched in heart muscle.
+To do so, a filter is applied on the expression value provided by HPA and measured in TPM (last column of the output file).
+In ProteoRE we'll use the "Filter by keywords and/or numerical value" tool.
+
+
+> ### {% icon hands_on %} Hands-on: Filter on expression value criterium
+>
+> 1. **Filter by keywords and/or numerical value** {% icon tool %} with the following parameters:
+>    - *"Operation"*: `Discard`
+>    - In *"Filter by numerical value"*:
+>        - {% icon param-repeat %} *"Insert Filter by numerical value"*
+>            - *"Column number on which to apply the filter"*: `c4`
+>            - *"Select operator"*: `<=`
+>            - *"Value"*: `10.0`
+>    - *"Sort by column ?"*: `Yes`
+>
+>
+> > ### {% icon question %} Questions
+> >
+> > How many lines are there in the file of heart transcripts
+> > with a TPM value >10 ?
+> >
+> > > ### {% icon solution %} Solution
+> > > 5257 lines.
+> > {: .solution}
+> {: .question}
+>
+>
+{: .hands_on}
+
+We have now 2 datasets of heart-muscle proteins/transcripts, based on IHC data or TPM value.
+
+We want now to select candidate biomarkers that are expressed in the heart muscle according to **both** IHC and RNA-seq data, using the Jvenn tool.
+
+
+> ### {% icon hands_on %} Hands-on: Venn diagram
+>
+> 1. **Venn diagram** {% icon tool %} with the following parameters:
+>    - In *"List to compare"*:
+>        - {% icon param-repeat %} *"Insert List to compare"*
+>            - *"Enter your list"*: `Input file containing your list`
+>                - *"Enter the name of this list"*: `heart IHC`
+>        - {% icon param-repeat %} *"Insert List to compare"*
+>            - *"Enter your list"*: `Input file containing your list`
+>                - *"Enter the name of this list"*: `heart RNAseq`
+>
+>   > ### {% icon question %} Questions
+>   > How many IDs are in common to both IHC and RNA-seq lists ?
+>   >   > ### {% icon solution %} Solution
+>   >   > 931 (you can visualize it in [the output of the Venn](#figure-1))
+>   > {: .solution}
+>   {: .question}
+{: .hands_on}
+
+You can see the the graphical output of the Venn:
+
+![Venn diagram output](../../images/jVenn_chart-tuto2.png)
```

Comment:
I added the Venn output image, maybe you would like to add some discussion of the different things shown (and what they tell us about our data)